ts/components/MemberListItem.tsx

```diff
@@ -174,7 +174,7 @@ function localisedStatusFromMemberStatus(memberStatus: MemberStateGroupV2) {
     case 'REMOVED_UNKNOWN': // fallback, hopefully won't happen in production
     case 'REMOVED_MEMBER': // we want pending removal members at the end of the "invite" states
     case 'REMOVED_MEMBER_AND_MESSAGES':
-      return null; // no text for those 3 pending removal states
+      return window.i18n('groupPendingRemoval'); // no text for those 3 pending removal states
```

Comment:
There is now text so should the comment `// no text for those 3 pending removal states` be updated?